src/vs/workbench/services/workspaces/browser/abstractWorkspaceEditingService.ts

```diff
@@ -62,7 +62,7 @@ export abstract class AbstractWorkspaceEditingService extends Disposable impleme
 			availableFileSystems.unshift(Schemas.vscodeRemote);
 		}
 		let workspacePath = await this.fileDialogService.showSaveDialog({
-			saveLabel: mnemonicButtonLabel(localize('save', "Save")),
+			saveLabel: mnemonicButtonLabel(localize('save', "Save")).withMnemonic,
```

Comment:
We can probably remove the use of `mnemonicButtonLabel` here as there is no `&` in the label `Save`